homeassistant/components/shelly/const.py

```diff
@@ -64,18 +64,31 @@
 # List of battery devices that maintain a permanent WiFi connection
 BATTERY_DEVICES_WITH_PERMANENT_CONNECTION: Final = ["SHMOS-01"]
 
+# Click events for Block devices
 EVENT_SHELLY_CLICK: Final = "shelly.click"
 
+# Button events for RPC devices
+EVENT_SHELLY_BUTTON: Final = "shelly.button"
```

Comment:
Why can't we just put it all in the same event ?

Reply:
I have pushed a commit which uses single event. Beside that `btn_up` and `btn_down` less fit for an event named `shelly.click` it is not that bad. Events are not overlapping so they can still share the same base `shelly.click` event.

Reply:
Offline discussion: Everyone is ok to go with single `shelly.click` event